start.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+from http.server import HTTPServer, SimpleHTTPRequestHandler, HTTPStatus
+import ssl
+from os import path, system
+
+# Generate private key and self-signed cert if it doesn't exist
+if not path.isfile('development.pem'):
+  system('/bin/bash -c "openssl req -new -x509 -keyout development.pem -out development.pem -days 365 -nodes -subj /CN=localhost/ -reqexts SAN -extensions SAN -config <(cat /etc/ssl/openssl.cnf <(printf \'[SAN]\nsubjectAltName=DNS:localhost\'))"')
```

Comment:
I'm assuming that this won't work on Windows--suggest updating the readme to include details about the need for a ``development.pem`` & maybe a guide on how to generate that by hand.

Reply:
I added a few options for windows users.  I think it would be a good idea to allow the dashboard to operate properly without web crypto too.  It's just the user will need to supply their PAT on each page load.  That's an option for development for those without OpenSSL or who don't want to deal with localhost https challenges.

Reply:
These look good--thanks!